File: WebViewExample/app/src/main/java/com/example/webviewexample/MainActivity.java
package com.example.webviewexample;

import android.os.Bundle;
import android.webkit.WebView;
import android.webkit.WebViewClient;

import androidx.appcompat.app.AppCompatActivity;


public class MainActivity extends AppCompatActivity {
    WebView webView;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
      setContentView(R.layout.activity_main);
      webView=findViewById(R.id.wv);
      webView.setWebViewClient(new WebViewClient());
      webView.loadUrl("httpS://google.com");
    }
}

File: Android task/TASK2/app/src/main/java/com/example/task2/MainActivity.kt
package com.example.task2

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_main)

        val secondActbutton = findViewById<Button>(R.id.second_act_btn)
        secondActbutton.setOnClickListener {
            val Intent = Intent(this,SecondActivity :: class.java)
            startActivity(Intent)

        }
    }
}